common/bet.ts

```diff
@@ -35,4 +38,21 @@ export type NumericBet = Bet & {
   allBetAmounts: { [outcome: string]: number }
 }
 
+// Binary market limit order.
+export type LimitBet = Bet & {
+  orderAmount: number // Amount of limit order.
+  limitProb: number // [0, 1]. Bet to this probability.
+  isFilled: boolean // Whether all of the bet amount has been filled.
+  isCancelled: boolean // Whether to prevent any further fills.
```

Comment:
"canceled" is the preferred American spelling

Reply:
Oh, I guess I can fix that then. Maybe after my tests with discord people so as not to break things. Or, maybe I'll just leave it haha. Two L's makes more sense to me.